### PR TITLE
add missing translationRef export necessary for internationalization

### PR DESCRIPTION
**Pull request by gabrielcbe**
### Summary 📰

In the previous [PR](https://github.com/procore-oss/backstage-plugin-announcements/pull/497) we made to add adds support for internationalization, we forgot to add necessary export to the translationRef to be used by people using the plugin

### Details 📚

added the necessary exports to the plugins to allow the intl to be used and translated inside their backstage instance

#### Checklist ✅

- [x] I have updated the necessary documentation
- [x] I have signed off all my commits as required by [DCO](https://GitHub.com/apps/dco/)
- [x] My build is green

### CODE REVIEW

File: plugins/announcements/src/index.ts [hunk 1]
```diff
@@ -15,3 +16,8 @@ export type AnnouncementsApi = AnnouncementsApi_;
  * @deprecated Use `announcementsApiRef` from `@procore-oss/backstage-plugin-announcements-react` instead
  */
 export const announcementsApiRef = announcementsApiRef_;
+
+/**
+ * @public
+ */
+export const announcementsTranslationRef = translationRef_;
```
Comment (kurtaking) [inline plugins/announcements/src/index.ts:23]: Revert this change, and things look good to me. The other exports were migrated to `announcements-react` and are marked as `@deprecated`.

**Review by kurtaking — CHANGES_REQUESTED** [review]

File: plugins/announcements/src/index.ts (hunk 1 above)
Comment (gabrielcbe) [reply plugins/announcements/src/index.ts:23]: Do you mean to revert the 'public' part, or the full export?

If its to revert the export too, I'd have to install locally the announcements-react too to use the translation, doing it here, i could install only the @procore-oss/backstage-plugin-announcements and be able to translate.

Comment (kurtaking) [reply plugins/announcements/src/index.ts:23]: > Do you mean to revert the 'public' part, or the full export?
> 

The full export. It needs to come from `announcements-react`. 

> If its to revert the export too, I'd have to install locally the announcements-react too to use the translation

Installing `announcements-react` to use this is the right path though. It's the recommended way from Backstage and its how they have their core plugins designed. Take a look at the catalog or scaffolder plugins. You will notice the same separation of concerns between `catalog`, `catalog-react`, `catalog-common`, etc. 

> doing it here, i could install only the @procore-oss/backstage-plugin-announcements and be able to translate.

Yes, but the export comes from the `announcements-react` plugin, so that others can use it without needing to rely on the `announcements` frontend plugin.

Comment (kurtaking) [reply plugins/announcements/src/index.ts:23]: @gabrielcbe - just made this realization. Aren't you supposed to use the `useAnnouncementsTranslation` hook directly?

Comment (gabrielcbe) [reply plugins/announcements/src/index.ts:23]: Hey, I'll make these changes tomorrow when I have my computer on me.
And as for the later question, we don't use the hook directly, in our backstage instance where the announcements is installed, we get the translationRef and use it to pass it the values in Portuguese (in our case) for each key in the translationRef that comes from your plugin. Here are the docs, https://backstage.io/docs/plugins/internationalization/

Comment (gabrielcbe) [reply plugins/announcements/src/index.ts]: @kurtaking, just pushed a fix removing the export.
Also renamed the translationRef for a more explicit name for announcements plugin

**Comment by kurtaking** [comment]
@gabrielcbe - add and changeset and we can merge this.

**Comment by gabrielcbe** [comment]
@kurtaking, added back the changeset, the one before had changes to the main plugin too so I reverted.

thanks a lot!

File: .changeset/tricky-pianos-peel.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@procore-oss/backstage-plugin-announcements-react": patch
+---
+
+add missing translationRef export necessary for internationalization
```
Comment (kurtaking) [inline .changeset/tricky-pianos-peel.md:5]:
```suggestion
add missing announcementsTranslationRef export necessary for internationalization
```

**Review by kurtaking — CHANGES_REQUESTED** [review]
last, last thing @gabrielcbe 😅
